examples/cdn-assets/cdn.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express')
+const path = require('path')
+
+const NEXT_DIR = path.join(__dirname, '.next')
+
+const app = express()
+// `-` is the default build ID. A real CDN would use a version number/etc here.
+app.use('/assets/-/', express.static(NEXT_DIR))
```

Comment:
Not hard, necessarily, but I don't think it's worth the effort. In production, I'll be using CloudFront, not a contrived node script.

Thoughts?